pkg/sql/oidext/oidext.go

```diff
@@ -16,6 +16,11 @@ package oidext
 
 import "github.com/lib/pq/oid"
 
+// CockroachPredefinedOIDMax defines the maximum OID allowed for use by
+// non user defined types. OIDs for user defined types will start at
+// CockroachPrefixedOIDMax and increase as new types are created.
+const CockroachPredefinedOIDMax = 100000
```

Comment:
I'm not sure. Last time I looked, I saw that Postgres just has an oid pool that it takes from -- it will query the the target pg_catalog table when creating a new oid to see if it is not present before returning a new oid.

Reply:
Do we need to worry that will give us headaches once we have oid pools?

Reply:
I don't think so -- I left a comment where we allocate the table ID. If are in the space were allocated ID + this offset overflows, then we will just grab a new id from the descriptor free list (once that exists). If a new descriptor ID exists, then the corresponding type ID is free real estate.